src/components/molecules/table/index.tsx

```diff
@@ -283,7 +283,7 @@ export const TablePagination = ({
     >
       <div>{`${offset + 1} - ${pageSize} of ${count} ${title}`}</div>
       <div className="flex space-x-4">
-        <div>{`${currentPage + 1} of ${pageCount}`}</div>
+        <div>{`${offset / 14} of ${pageCount}`}</div>
```

Comment:
I think this causes some weird errors in the pagination component where the first page is 0 and the subsequent pages all have rounding errors. I dont know what the correct solution to this would be but doesn't the `currentPage + 1` work?

![image](https://user-images.githubusercontent.com/88927411/152490098-025c8f58-ae6c-4f2c-8123-cfd068a7d051.png)
![image](https://user-images.githubusercontent.com/88927411/152490116-95c6e102-90da-47af-928d-eaa70a637b10.png)
![image](https://user-images.githubusercontent.com/88927411/152490152-353eb6da-3f3c-4c68-bf58-9fee7ca71806.png)